sssom/cliques.py

```diff
@@ -18,41 +18,42 @@ def to_networkx(msdf: MappingSetDataFrame) -> nx.DiGraph:
     # m = {
     #    "owl:subClassOf",
     # }
-    for mapping in doc.mapping_set.mappings:
-        s = mapping.subject_id
-        o = mapping.object_id
-        p = mapping.predicate_id
-        # TODO: this is copypastad from export_ptable
-
-        pi = None
-
-        if p == "owl:equivalentClass":
-            pi = 2
-        elif p == "skos:exactMatch":
-            pi = 2
-        elif p == "skos:closeMatch":
-            # TODO: consider distributing
-            pi = 2
-        elif p == "owl:subClassOf":
-            pi = 0
-        elif p == "skos:broadMatch":
-            pi = 0
-        elif p == "inverseOf(owl:subClassOf)":
-            pi = 1
-        elif p == "skos:narrowMatch":
-            pi = 1
-        elif p == "owl:differentFrom":
-            pi = 3
-        elif p == "dbpedia-owl:different":
-            pi = 3
-
-        if pi == 0:
-            g.add_edge(o, s)
-        elif pi == 1:
-            g.add_edge(s, o)
-        elif pi == 2:
-            g.add_edge(s, o)
-            g.add_edge(o, s)
+    if doc.mapping_set.mappings is not None:
+        for mapping in doc.mapping_set.mappings:
+            s = mapping.subject_id
+            o = mapping.object_id
+            p = mapping.predicate_id
+            # TODO: this is copypastad from export_ptable
+
+            pi = None
```

Comment:
what happened to all of that hard work on improving the variable names here?

Reply:
Any idea @matentzn ? I remember the refactoring of all these in a different branch. I thought merging down from `main` would have reflected those changes. Also I just worked off of the branch you had @cthoyt .

Reply:
you can see it's because the code was copy-pasted, and I only fixed it in one place. leave it for now

Reply:
Maybe cliques.py got a chris PR in the meantime. Unless PR howls at bad variable names, we should expect those coming back from time to time.

Reply:
yeh I agree this is horrible... but I wouldn't block this PR on it. We should take my code factor it into the logic (mapping mappings to logical axioms) and then the ptable-format specific stuff. I will be better in future.

Reply:
Agreed we will fix this in a future refactoring PR

Reply:
See #152